File: 30-days-of-code/day-3---intro-to-conditional-statements/solution.py
#!/bin/python3

import sys


N = int(input().strip())

def isWeird(num):
    if num % 2 != 0:
        return 'Weird'
    if num >= 2 and num <= 5:
        return 'Not Weird'
    if num >= 6 and num <= 20:
        return 'Weird'
    if num > 20:
        return 'Not Weird'

print(isWeird(N))

File: 30-days-of-code/day-3---intro-to-conditional-statements/solution.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;
public class Solution {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        scan.close();

        System.out.println(isWeird(n));
    }

    public static String isWeird(int num) {
        if (num % 2 != 0) return "Weird";
        if (num >= 2 && num <= 5) return "Not Weird";
        if (num >= 6 && num <= 20) return "Weird";
        return "Not Weird";
    }
}
